Backend/src/main/java/com/example/SkyLine/DTO/NotificationRequestDTO.java

```diff
@@ -0,0 +1,112 @@
+package com.example.SkyLine.DTO;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+public class NotificationRequestDTO {
+
```

Comment:
put attributes before their getters and setters and for getters and setters you can use lombook for simplicity